### PR TITLE
feat: responsify history funding payments table

### LINKED ISSUE #753: [DEX] Funding payment history mobile responsiveness
### **Design**
- [Funding payment history](https://www.figma.com/file/qJtXzfX9nzh0c7yA6G9iI3/Injective-DEX?node-id=4217%3A10896)

### **Summary**
- [ ] Users can see funding payment amount, payment timestamp and the corresponding market pair in one single view for each funding payment history.

### PR DESCRIPTION
**Pull request by ThomasRalee**
## This PR:
- resolves #753 
- figma [design](https://www.figma.com/file/qJtXzfX9nzh0c7yA6G9iI3/Injective-DEX?node-id=4217%3A10896)

## Screenshot:
image

### CODE REVIEW

File: components/partials/common/derivatives/mobile-funding-payment.vue [hunk 1]
```diff
@@ -0,0 +1,150 @@
...
+
+    total(): BigNumberInBase {
+      const { market, fundingPayment } = this
+
+      if (!market || !fundingPayment.amount) {
+        return ZERO_IN_BASE
+      }
+
+      return new BigNumberInWei(fundingPayment.amount).toBase(
+        market.quoteToken.decimals
```
[the remark this answers is not in the document]
Comment (shane-moore) [reply components/partials/common/derivatives/mobile-funding-payment.vue:123]: spent some time this weekend reading about how/why we convert decimal places, so I can mark this as resolved

**Review by shane-moore — APPROVED** [review]

**Review by afterburn — APPROVED** [review]

**Review by bangjelkoski — APPROVED** [review]

**Comment by Tofel** [comment]
Looks good ✅ 

Just one question about sorting... how is it realised now? If I had 2 positions opened for 2 different pairs I'd expect the funding payments to be displayed more or less like that:
- funding for pair A
- funding for pair B
- funding for pair A
- funding for pair B

But what I actually saw was:
image

**Comment by Tofel** [comment]
@terencetang9 yeah, shouldn't block. Let's create an enhancement issue to sort by timestamp and if that's equal, then sort by ticker?

**Review by Tofel — APPROVED** [review]

**Comment by terencetang9** [comment]
@Tofel yes will do!

**Review by bangjelkoski — APPROVED** [review]

**Pull request merged**